scripts/casestudies/schema.json

```diff
@@ -0,0 +1,281 @@
+{
+	"id":"adeogroup",
+	"company":{
+		"name":"Adeo Group",
+		"description":"Adeo owns different brands in retail industry focused on home improvement and DIY markets, like Leroy Merlin.",
+		"customers":"500M",
+		"industry":"Retail",
+		"revenue":"25.6B EURO turnover, including 768M EURO online.",
+		"website":"https://www.adeo.com/",
+		"logo":"/img/casestudies/adeo/logo.svg",
+		"contact":[
+			{
+				"name":"Ludovic Dussart",
+				"link":"https://twitter.com/ldussart"
+			}
+		]
+	},
+	"challenges":"Cost Component Repository product, part of the ADEO tech products, is used to calculate and publish transfer prices between different internal locations on the globe. Different business units use different information systems. It is hard to learn how each business unit share information about their systems, their API and accuracy.
+
+Initial solution was a developer portal with list of all applications and reference to dedicated documentation. Some legacy systems had docs written in MS Excel.
+
+There was a need for a standart way of describing event-driven architecture.
+",
```

Comment:
@derberg Just a small query, will we be allowing multiple strings inside JSON in this way? Because this error is not a es-lint error (which we can ignore for a bit), but a typical JSON error.

Reply:
I'll fix it, issue with initial tool I used to generate basic schema, it should not do this but use line break character. Thanks!